go.mod

```diff
@@ -0,0 +1,44 @@
+module github.com/GoogleCloudPlatform/kubectl-ai
+
+go 1.23.2
+
+replace github.com/GoogleCloudPlatform/kubectl-ai/gollm => ./gollm
+
+require github.com/GoogleCloudPlatform/kubectl-ai/gollm v0.0.0-00010101000000-000000000000
```

Comment:
curious why this is needed and if it is it might be good to leave a comment for posterity.